aibolit/ast_framework/java_class_decomposition.py

```diff
@@ -20,14 +20,48 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from typing import List, Dict, Set, Iterator
+from typing import List, Dict, Set, Iterator, Any
 
 from networkx import DiGraph, strongly_connected_components, weakly_connected_components  # type: ignore
 
 from aibolit.ast_framework import AST, ASTNodeType
+from aibolit.patterns.classic_getter.classic_getter import ClassicGetter as getter
 
 
-def decompose_java_class(class_ast: AST, strength: str) -> List[AST]:
+def find_patterns(tree: AST, patterns: List[Any]) -> Set[str]:
+    """
+    Searches all setters in a component
+    :param patterns: list of patterns to check
+    :param tree: ast tree
+    :return: list of method name which are setters
+    """
+
+    setters: Set[str] = set()
```

Comment:
It is better be named like `patterns_method_names`.

Reply:
i can't understand what it means. What do you mean by `patterns_method_names`